tests/game_selection_test.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import mock
+from mock import patch
+from mock import Mock
+from ..lib.ui import UI
+from ..lib.game_selection import *
+
+
+class TestGameSelections:
```

Comment:
Could you add the following tests?

1. Test that the `ui` got called with the invalid input message
2. Test that returns a list of two human players given an input `h`
3. Test that returns a list of a human player and computer player given an input `c`

Reply:
done